lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -2459,7 +2459,7 @@ namespace Js
             Var member;
 
             // If the item is found in the array own body, then it is a number
-            if (JavascriptOperators::GetOwnItem(object, index, &member, scriptContext))
+            if (JavascriptOperators::GetOwnItem(object, index, &member, scriptContext) && member != scriptContext->GetLibrary()->GetUndefined())
```

Comment:
As far as I now, typed arrays can't contain `undefined`. Instead of checking if the value is `undefined` here, I think `JavascriptOperators::GetOwnItem` should return `false` when the item doesn't exist.

It might be OK to keep it the way it currently is for the `1.2` milestone and look into returning `false` later since `GetItem` (called by `GetOwnItem`) is used in a bunch of places. @abchatra, what do you think?